plugins/generic/customBlockManager/CustomBlockPlugin.inc.php

```diff
@@ -176,13 +183,29 @@ function manage($verb, $args) {
 	 */
 	function getContents(&$templateMgr) {
 		$journal =& Request::getJournal();
+		$journalId = $journal->getId();
+		
 		if (!$journal) return '';
+		
+		$locale = AppLocale::getLocale();
+		$primaryLocale = AppLocale::getPrimaryLocale();
+		$blockContent = $this->getSetting($journalId, 'blockContent');
+		$blockContentTxt = '';
```

Comment:
(I'd suggest adding an upgrade step so that content is standardized to the new localized form during the upgrade, so we don't have to consider both cases in the future.)